source/fundamentals/connection.txt

```diff
@@ -0,0 +1,510 @@
+================
+Connection Guide
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. toctree::
+
+   /fundamentals/connection/tls
+
+This guide shows you how to connect to a MongoDB instance or replica set
+deployment using the driver. For information about authenticating
+with a MongoDB instance, see :ref:`Authentication <authentication>`.
+
+.. _mongoclient:
+
+MongoClient
+-----------
+
+Use the ``MongoClients.create()`` method to construct a
+``MongoClient``. As each ``MongoClient`` represents a threadsafe *pool* of
+connections to the database, most applications only require a single
+instance of a ``MongoClient``, even across multiple threads. All resource
+usage limits, such as max connections, apply to individual
+``MongoClient`` instances.
+
+.. note::
+
+   Always call ``MongoClient.close()`` to clean up resources when an
+   instance is no longer needed.
```

Comment:
Good to mention. Since our examples use try with resources, I wonder if that might be appropriate to mention as well as an alternative to close(), and link to: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html